site.go

```diff
@@ -63,6 +65,14 @@ type Site struct {
 	SkipIndexURLPattern *regexp.Regexp
 }
 
+func (s *Site) GetResources(dir, version string) (http.FileSystem, error) {
+	c, err := s.Content.OpenVersion(context.Background(), version)
+	if err != nil {
+		return nil, err
+	}
+	return &subdirFileSystem{fs: c, path: "_resources/" + dir}, nil
```

Comment:
Might want to use `filepath.Join` here unless there's a reason unix filepaths are always alright (I think this is true for go-embed, for example).